docs/v3_services.rst

```diff
@@ -23,49 +29,47 @@ such as:
 
   http://secrdrop5wyphb5x.onion/
 
-For new SecureDrop instances, it is recommended to
-enable v3 onion services only. For existing instances, migration
-to v3 onion services is recommended, and the ability to run both
-v2 and v3 onion services simultaneously is supported to make the process
-easier.
+.. important::
+
+   If you are currently advertising a 16-character v2 address like the above
+   to your sources, it will become unreachable in February 2021. You must
+   update to v3 onion services before then.
+
+   There is no downgrade path from v3 to v2 using the ``securedrop-admin``
+   tool. We recommend that you follow the v2+v3 migration path below, and test v3
+   functionality thoroughly before disabling
+   v2 services.
```

Comment:
could you link "disabling v2 services" to a section in the doc that says how to do this? right now we just have a section on how to enable v3 onion service which somewhere in the middle says:

> Next, enable v3 onion services (and optionally disable v2 services) using:
> ./securedrop-admin sdconfig

Reply:
Thanks @creviera, addressed as part of #5379.